### ISSUE TITLE
Web support on 2.0.0

### ISSUE THREAD

**Issue by SimoGh** [flutterdata/flutter_data#288]
Hi,

Looks like using 2.0.0 running on web is not working, I get the following error :
`sqlite3-2.4.6/lib/src/ffi/api.dart:1:8: Error: Dart library 'dart:ffi' is not available on this platform.`

Is this expected ?

**Comment by frank06** [comment]
Yes, there are no plans to fix web. I would like to, but, it is not a priority
